spec/registry.dtd

```diff
@@ -1,4 +1,7 @@
 <!ELEMENT registry (function|validationRule)*>
+<!ATTLIST registry
+    xml:lang       NMTOKEN  #IMPLIED
+>
```

Comment:
I added `xml:lang` as an optional attribute on the top-level `<registry>`, as well as directly on `<input>` and `<option>`, to indicate the language of the included descriptions.

Should this be required, though? Or are there use cases where a `<registry>` could be e.g. embedded in another XML document, where the locale is defined somewhere further up the tree?